app/src/main/java/com/example/musicapp/data/source/local/dao/ResolverDataImp.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.musicapp.data.source.local.dao
+
+import android.annotation.SuppressLint
+import android.content.ContentResolver
+import android.content.Context
+import android.provider.MediaStore
+import com.example.musicapp.data.model.Artist
+import com.example.musicapp.data.model.Song
+import com.example.musicapp.data.source.repository.MusicDataSource
+import com.example.musicapp.ui.main.MainActivity
+
+class ResolverDataImp(private var contentResolver: ContentResolver) : ResolverDataDao {
```

Comment:
singleton private constructor nhé 
```suggestion
class ResolverDataImp private constructor (private var contentResolver: ContentResolver) : ResolverDataDao {
```

Reply:
Oki a

Reply:
Vâng a